feature-libs/cart/base/styles/components/_cart-totals.scss

```diff
@@ -5,12 +5,8 @@
 
   @include media-breakpoint-down(md) {
     padding-top: 2rem;
-    padding-inline-end: 3rem;
```

Comment:
Did you intentionally remove the following lines?

Reply:
yes

Reply:
This one shouldn't be in here. I'll remove it.